rasa/core/tracker_store.py

```diff
@@ -991,7 +993,7 @@ def _create_database(engine: "Engine", database_name: Text) -> None:
         conn.close()
 
     @contextlib.contextmanager
-    def session_scope(self):
+    def session_scope(self) -> Generator["sessionmaker", None, None]:
```

Comment:
this yields a `Session`, doesn't it? `sessiomaker()` returns a `Session` as far as I know

Reply:
Yes, you're right, it was initially Session before I knew I had to put it in `" "` 😄 , I'll change it now